internal/services/network/public_ip_prefix_resource_test.go

```diff
@@ -116,6 +116,9 @@ func TestAccPublicIpPrefix_prefixLength31(t *testing.T) {
 func TestAccPublicIpPrefix_prefixLength24(t *testing.T) {
 	// NOTE: This test will fail unless the subscription is updated
 	//        to accept a minimum PrefixLength of 24
+	// more detail about [public ip limits](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-addresses#limits)
+	// you can submit a support request to increase the limit in [Azure Portal](https://learn.microsoft.com/en-us/azure/networking/check-usage-against-limits#azure-portal)
+	t.Skip("skip for test subscription's `Public IPv4 Prefix Length` is 28")
```

Comment:
```suggestion
	t.Skip("skip for test subscription's `Public IPv4 Prefix Length` is 28")
```

test subscription has been udpated

Reply:
This case starts failing days after the subscription update. could you please check on the subscription's current status?

![1668567565(1)](https://user-images.githubusercontent.com/2633022/202072871-0be30802-36c8-4e1e-852c-f29ade88e841.jpg)